### ISSUE TITLE
Mixed ores everywhere setting

### ISSUE THREAD

**Issue by h8ul-modder** [h8ul-modder/factorio-ribbon-maze#28]

**Comment by OvermindDL1** [comment]
Sounds horrifying, but I'd try it on challenge builds.  ^.^

**Comment by OvermindDL1** [comment]
Although you should prevent it from generating mixed ores that take different fluid types to harvest or it will make some impossible to get (especially if infinite).

**Comment by h8ul-modder** [comment]
Agreed! I was lazy and only made it apply only to resources that are mixed anyway (i.e. starting resources) so it should avoid any problems... And keeps some resources rare for added cruelty :evilgrin:
